serverless/knative_serving/serverless-traffic-management.adoc

```diff
@@ -0,0 +1,27 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-traffic-management"]
+= Traffic management
+include::modules/common-attributes.adoc[]
+:context: serverless-traffic-management
+
+toc::[]
+
+// TODO later: consider splitting this into a conceptual module about revisions; would need to plan where these conceptual topics would be included in docs
+A revision is a point-in-time snapshot of the code and configuration for each modification made to a Knative service. Each time the configuration of a service is updated, a new revision for the service is created. Revisions are immutable objects and can be retained for as long as they are required or used. Knative Serving revisions can be automatically scaled up and down according to incoming traffic.
```

Comment:
If this is the first time that "revision" is mentioned, maybe "\_revision\_" to emphasize the first use as mentioned in the [quick markup reference](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference).

Nice crisp definition, btw.

Reply:
Done, and thank you :)